packages/payload/src/auth/operations/refresh.ts

```diff
@@ -57,7 +57,7 @@ async function refresh(incomingArgs: Arguments): Promise<Result> {
     },
   } = args
 
-  if (typeof args.token !== 'string') throw new Forbidden(args.req.t)
+  if (typeof args.token !== 'string' || args.req.user == null) throw new Forbidden(args.req.t)
```

Comment:
I think something to handle all falsy values would be better. Something like `|| !args.req.user`